orchestrator/src/main/kotlin/Orchestrator.kt

```diff
@@ -114,8 +112,7 @@ class Orchestrator(
         db.blockingQueryCatching(transactionIsolation = isolationLevel) {
             val ortRun = getCurrentOrtRun(configWorkerResult.ortRunId)
 
-            val context = WorkerScheduleContext(ortRun, workerRepositories, publisher, header, emptyMap())
-            context to listOf(createAnalyzerJob(ortRun).let { { scheduleAnalyzerJob(ortRun, it, header) } })
+            nextJobsToSchedule(ConfigEndpoint, ortRun.id, header, jobs = emptyMap())
```

Comment:
Not directly related to this change, but in my opinion we should also create jobs for the config worker to get rid of the requirement for the special handling. This would also allow to handle it consistently in other places, e.g. for showing that the config worker is currently running when the status of an ORT run is requested via API, or to also measure the activity of the config worker when exporting stats to Graphite.

Reply:
I would rather defer this for now; this can be considered together with other changes on the DB in the context of deleting ORT runs.

Reply:
Yes, I didn't mean to change it now, just wanted to express the idea.